src/app/cube/home/home.component.scss

```diff
@@ -18,7 +18,7 @@
     width: 100%;
     height: 100vw;
     z-index: 1;
-    background: white;
+    background: #f6f9fc;
```

Comment:
Do we have a variable for this?

Reply:
No, this is remanent of before we had a global sass variable for almost every color on our site.  I can add it if you want.

Reply:
Let's use the variables. This makes it easier if we ever need adjust colors for some reason. This way we only have to update the white color we use in one place versus the 100+ stylesheets we have in the client.